README.md

```diff
@@ -151,6 +146,48 @@ await wallet.syncSendMessage({
 
 It is only possible to perform sync calls within the confines of one shard.
 
+## Multi-currency and bouncing
+
+=nil; provides a multi-currency mechanism. A contract can be the owner of one custom currency, and owners can freely send custom currencies to other contracts. As a result, the balance of a given contract may contain standard =nil; tokens, and several custom tokens created by other contracts.
```

Comment:
@KlonD90 wdyt maybe not mention `=nil; tokens` and just say `standard tokens`?

Reply:
because it's not like standard token but our standard

Reply:
I suggest to omit '=nil;', it looks like the name of the token, isn't it?

Reply:
Got it! Removed =nil;